backends/ebpf/tests/ptf/common.py

```diff
@@ -44,6 +46,14 @@
 TEST_PIPELINE_MOUNT_PATH = "/sys/fs/bpf/pipeline{}".format(TEST_PIPELINE_ID)
 PIPELINE_MAPS_MOUNT_PATH = "{}/maps".format(TEST_PIPELINE_MOUNT_PATH)
 
+PORT0 = 0
+PORT1 = 1
+PORT2 = 2
+PTF_PORTS = [PORT0, PORT1, PORT2]
```

Comment:
If `DP_PORTS` is described I think that `PTF_PORTS` (and `PORTx`) also should have small description. The most interesting thing for me is the difference between these constant values (e.g. when which variable should be used because this might be confusing).